DatasetLoader.py

```diff
@@ -22,19 +22,20 @@ def download_and_extract_zip(zip_file_path, extract_path):
 
 
 def data_loader(dataset_path, image_size, batch_size, channels, dataset_dir="./datasets"):
-
     normalization_args = list((0.5 for _ in range(channels)))
 
     transform = transforms.Compose(
         [transforms.Resize(image_size),
-            transforms.ToTensor(),
+         transforms.CenterCrop(image_size),
+         transforms.ToTensor(),
          transforms.Normalize(normalization_args, normalization_args)
          ])
 
     if dataset_path in VISION_DATASETS.keys():
         dataset = VISION_DATASETS[dataset_path]
         dataset = dataset(root=dataset_dir, train=True,
                           transform=transform, download=True,)
+        model_name = "mnist"
```

Comment:
```suggestion
        model_name = dataset_path
```
because we know dataset_path is one of the keys in VISION_DATASETS here.